app/controllers/api/users.js

```diff
@@ -63,3 +63,11 @@ exports.signup = function (req, res) {
     });
   });  
 };
+
+exports.currentUser = function (req, res) {
+  console.log(req.user);
+  if(req.user){
+   return res.send(req.user);
+  }
+  res.status(404).send('Not Found!');
+};
```

Comment:
I thought we added the newline prompt to our linter, please add a new line at the end of the file.